src/adapter/src/coord/ddl.rs

```diff
@@ -445,11 +452,29 @@ impl Coordinator {
     pub(crate) fn drop_compute_sinks(&mut self, sinks: impl IntoIterator<Item = ComputeSinkId>) {
         let mut by_cluster: BTreeMap<_, Vec<_>> = BTreeMap::new();
         for sink in sinks {
+            // Filter out sinks that have already been dropped.
+            //
+            // Note: Ideally we'd use .filter(...) on the iterator, but that would
+            // require simultaneously getting an immutable and mutable borrow of self
```

Comment:
Somewhere in this function we should add a comment explaining that the sink is not removed from `active_subscribes` until we receive a remove message from compute which is why we need the `dropping` flag.

Reply:
Good idea, I extended this comment to include an explanation